docs/docs/coverage/language/python.md

```diff
@@ -26,7 +26,7 @@ The following table provides an outline of the features Trivy offers.
 |-----------------|------------------|:-----------------------:|:----------------:|:------------------------------------:|:--------:|:----------------------------------------:|
 | pip             | requirements.txt |            -            |     Include      |                  -                   |    ✓     |                    ✓                     |
 | Pipenv          | Pipfile.lock     |            ✓            |     Include      |                  -                   |    ✓     |                Not needed                |
-| Poetry          | poetry.lock      |            ✓            |     Exclude      |                  ✓                   |    -     |                Not needed                |
+| Poetry          | poetry.lock      |            ✓            |     Include      |                  ✓                   |    -     |                Not needed                |
```

Comment:
link for `Include` (https://github.com/aquasecurity/trivy/pull/8134#discussion_r1896422468)

Reply:
I've added a link for all package managers in [this](https://github.com/aquasecurity/trivy/pull/8134) PR